client/src/components/performerHistory/HistoryCard.js

```diff
@@ -1,19 +1,16 @@
-import React, {Fragment} from 'react';
+import React, { Fragment } from 'react';
 import PropTypes from 'prop-types';
-import {Link} from 'react-router-dom'
+import { Link } from 'react-router-dom';
 
 const HistoryCard = (performances) => {
   return (
-    <Fragment>
```

Comment:
Why have we removed this from a fragment? They are supposed to show enhanced modularity and stop things from breaking other states